system/keymaps/keyboard.xml

```diff
@@ -52,8 +52,11 @@
       <pageup>PageUp</pageup>
       <pagedown>PageDown</pagedown>
       <return>Select</return>
+      <return mod="long">ContextMenu</return>
```

Comment:
Is it supposed to work inside <remote>...</remote> tags also?
Longpress mod seems completely ignored in remote.xml